rel/overlay/bin/remsh

```diff
@@ -71,6 +81,7 @@ if [ ! -z "$VERBOSE" ]; then
   set -x
 fi
 
-exec "$BINDIR/erl" -boot "$ROOTDIR/releases/$APP_VSN/start_clean" \
+# clear ERL_FLAGS to avoid conflicts with the cli parameters
```

Comment:
Is it possible that `ERL_FLAGS` might have other settings you *would* want to pass into `erl`, such as `-args_file` or `-pa`? The latter seems especially useful for e.g. `recon`, `weatherreport`, etc. that might be stored out of tree.